diffpdf.json

```diff
@@ -0,0 +1,14 @@
+{
+    "homepage": "http://soft.rubypdf.com/software/diffpdf",
+    "version": "2.1.3",
+    "license": "GPL-2.0",
+    "url": "http://soft.rubypdf.com/wp-content/uploads/2010/08/diffpdf-2.1.3-win32-static.zip",
+    "hash": "b698b9c3c3c0582c2b5ea474d4df0c5f864b649774e857fdc423b4949ef79b6d",
+    "bin": "diffpdf.exe",
+    "shortcuts": [
+        [
+            "diffpdf.exe",
+            "DiffPDF"
+        ]
+    ]
+}
```

Comment:
This could be added: 
````
    "checkver": {
        "re": ">DiffPDF ([\\d.]+) Windows 32-bit Static Version",
        "url": "http://soft.rubypdf.com/software/diffpdf"
    }
````